docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3'
+
+services:
+    catalog:
+        build:
+            context: catalog
+        ports:
+            - "5430:5432"
+        environment:
+            - POSTGRES_PASSWORD=testpassword
+
+    irods-catalog-provider:
+        build:
+            context: irods_catalog_provider
+        depends_on:
+            - catalog
+
+    irods-client-icommands:
+        build:
+            context: irods_client_icommands
+        depends_on:
+            - irods-catalog-provider
+
+    irods-client-rest-cpp:
+        build:
+            context: irods_client_rest_cpp
+            dockerfile: irods_client_rest_cpp.Dockerfile
+        volumes:
+            - ./irods_client_rest_cpp/irods_client_rest_cpp.json:/etc/irods_client_rest_cpp/irods_client_rest_cpp.json:ro
+        depends_on:
+            - irods-catalog-provider
+
+    nginx-reverse-proxy:
+        build:
+            context: irods_client_rest_cpp
+            dockerfile: nginx.Dockerfile
+        ports:
+            - "80:80"
+        depends_on:
+            - irods-client-rest-cpp
+
+    irods-client-zmt:
+        build:
+            context: irods_client_zone_management_tool
+        environment:
+            - REACT_APP_REST_API_URL=http://nginx-reverse-proxy/irods-rest/0.9.2
```

Comment:
TODO: Need to test to make sure this works when browsing using the host running running the containers. If it is not, I'm guessing `${hostname}` or `localhost` will work, but, again, only if the browser is on the same host as the containers.

Reply:
Confirmed